src/main/resources/org/jpeek/metrics/CCM.xsl

```diff
@@ -96,4 +147,23 @@ SOFTWARE.
       <xsl:apply-templates select="node()|@*"/>
     </xsl:copy>
   </xsl:template>
+  <xsl:template name="group">
+    <xsl:param name="x" as="element()"/>
+    <xsl:param name="seen"/>
+    <xsl:param name="graph"/>
+    <xsl:variable name="used" select="concat($seen, $x/@id)"/>
+    <xsl:for-each select="$x/edge">
+      <xsl:variable name="r" select="."/>
+      <xsl:if test="not(contains($used, $r/text()))">
```

Comment:
potential problem: You have a class with methods: _data_ and _dataFormatted_, contains return true for method _data_ if _dataFormatted_ is already visited.